airflow/models/connection.py

```diff
@@ -80,6 +80,7 @@
     "postgres": ("airflow.providers.postgres.hooks.postgres.PostgresHook", "postgres_conn_id"),
     "presto": ("airflow.providers.presto.hooks.presto.PrestoHook", "presto_conn_id"),
     "redis": ("airflow.providers.redis.hooks.redis.RedisHook", "redis_conn_id"),
+    "snowflake": ("airflow.providers.snowflake.hooks.snowflake.SnowflakeHook", "snowflake_conn_id"),
```

Comment:
Would you mind adding a simple test to check that all hooks' path are valid? For example using importlib?

Reply:
@turbaszek Added, but requires updating the MSSQL mapping to ODBC per deprecation notice. Since pymssql is deprecated, the existing hook throws an import error for the existing MSSQL mapping.

Reply:
Good! In this way, you've found a bug 👌